src/block/card.scss

```diff
@@ -0,0 +1,118 @@
+.card {
+  display: flex;
+
+  flex-direction: column;
+
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+
+  padding-left: 16px;
+  padding-right: 16px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    margin-top: 32px;
+    width: 160px;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    background-size: 100% 100%;
+  }
+
+  &__info {
+    margin: 0;
+    margin-top: 40px;
+    padding: 0;
+  }
+
+  &__name {
+    width: 100%;
+    height: 36px;
+    color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used here but it's not defined in this file. Ensure that it is defined or imported from another SCSS file.